app/src/org/commcare/activities/connect/ConnectIdBiometricConfigActivityUiController.java

```diff
@@ -44,7 +44,7 @@ public ConnectIdBiometricConfigActivityUiController(ConnectIdBiometricConfigActi
     @Override
     public void setupUI() {
         fingerprintButton.setOnClickListener(v -> activity.handleFingerprintButton());
-        pinButton.setOnClickListener(v -> activity.handlePinButton());
+//        pinButton.setOnClickListener(v -> activity.handlePinButton());
```

Comment:
We still need this button handler for when the user wants to (possibly) configure and proceed using a PIN as their screen unlock.